### PR TITLE
Regenerated IotHub Sdk

### PR DESCRIPTION
**Pull request by anusapan**
Regenerated SDK for the new swagger version (2019-07-01-preview).
Re-record the test session.
[Swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json)
[Swagger PR](https://github.com/Azure/azure-rest-api-specs/pull/7199)

### CODE REVIEW

File: sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj [hunk 1]
```diff
@@ -8,9 +8,9 @@
 		<Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
 		<AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
 		<AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-		<Version>2.9.0-preview</Version>
+		<Version>2.10.0-preview</Version>
 		<PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;</PackageTags>
-		<PackageReleaseNotes>Manual failover update for IotHub.</PackageReleaseNotes>
+		<PackageReleaseNotes>Regenerated IotHub Sdk.</PackageReleaseNotes>
```
Comment (isra-fel) [inline sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj]: This string ends up being displayed on nuget and viewed by the customer. Please make it more informative and user-friendly. Thanks

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hey @anusapan , when you update api version, please also update the api tag in `AzSdk.RP.props` under your solution folder. Thanks

File: sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj (hunk 1 above)
Comment (anusapan) [reply sdk/iothub/Microsoft.Azure.Management.IotHub/src/Microsoft.Azure.Management.IotHub.csproj]: String updated

**Comment by anusapan** [comment]
@isra-fel updated the api tag in `AzSdk.RP.props`

**Review by isra-fel — APPROVED** [review]

**Pull request merged**